### DISCUSSION TITLE
Why is the size of released TTS models smaller than the size of trained models ?

### DISCUSSION THREAD

**Discussion by mbarnig** [coqui-ai/TTS#1706] [General Q&A]
I noticed that the size of my best_models and checkpoints, when training or fine-tuning TTS-models, is bigger than those of the related released models.  

Here are two comparisons:    

### tts_models--de--thorsten--vits
* released model_file.pth : 332,6 MB
* my trained best_model_xxx.pth or checkpoint_yyy.pth : 997,7 MB

### tts_models--multilingual--multi-dataset--your_tts
* released model_file.pth : 380,0 MB
* my trained best_model_xxx.pth or checkpoint_yyy.pth : 1,0 GB

What is the reason for this difference ? I don't find any clues in the Coqui-TTS documentation.
How can I reduce the size of a best_model or a checkpoint for inference or for release ?

Thank you in advance for an answer.

**Comment by erogol** [comment]
YourTTS is a larger model. Larger layers.
